solutions/src/main/kotlin/de/thermondo/solutions/basics/Basics05Solution.kt

```diff
@@ -0,0 +1,18 @@
+package de.thermondo.solutions.basics
+
+/**
+ * Use the `class` keyword to define a class.
+ * Class naming is done in upper camel case.
+ *
+ * Create a class named Basics05
```

Comment:
```suggestion
 * Create a class named Basics05Solution
```

Let's make one more tiny adjustment here to nip any potential confusion in the bud